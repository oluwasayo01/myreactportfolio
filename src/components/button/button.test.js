import Button from '../button/button'
import renderer from 'react-test-renderer'
import React from 'react'
import { ThemeProvider } from 'styled-components'

test('Snapshot of a button element', () => {
    const tree = renderer.create(
        <ThemeProvider theme={{mode: "main"}}>
            <Button text="Hire Me" pressed="false"/>;
        </ThemeProvider>).toJSON()
        expect (tree).toMatchSnapshot();
    })

