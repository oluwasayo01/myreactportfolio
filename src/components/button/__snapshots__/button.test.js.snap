// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of a button element 1`] = `
Array [
  <button
    className="sc-AxjAm bUCjQH"
  />,
  ";",
]
`;
